File: C#/RBD/RBD/TPC/Msg/TimeoutMessage.cs
// + TODO check

using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

using RBD.Msg;
using RBD.Communication;

namespace RBD.TPC.Msg
{
    public class TimeoutMessage : Message
    {
        override
        public MessageType GetMessageType()
        {
            return MessageType.TPC_TIMEOUTMESSAGE;
        }

        override
        public void FromBinary(DataInputStream s) //throws IOException 
        {
            // empty
        }

        override
        public void ToBinary(DataOutputStream s) //throws IOException 
        {
            // empty
        }

    }
}

File: SR-RBD/src/ddb/tpc/msg/TimeoutMessage.java
package ddb.tpc.msg;

import java.io.IOException;

import ddb.communication.DataInputStream;
import ddb.communication.DataOutputStream;
import ddb.msg.Message;
import ddb.msg.MessageType;
/**
 * 
 * @author PKalanski
 * Wiadomosc reprezentujaca wystapienie timeout
 */
public class TimeoutMessage extends Message {

	@Override
	public MessageType getType() {
		// empty
		return MessageType.TPC_TIMEOUT_MESSAGE;
	}

	@Override
	public void fromBinary(DataInputStream s) throws IOException {
		// empty

	}

	@Override
	public void toBinary(DataOutputStream s) throws IOException {
		// empty

	}

}
